app.py

```diff
@@ -17,15 +20,27 @@
 
 from utils import force_git_push
 
-# These variables are for storing the mturk HITs in a Hugging Face dataset.
+
+def generate_respone(chatbot: ConversationChain, input: str) -> str:
+    """Generates a response for a `langchain` chatbot."""
+    return chatbot.predict(input=input)
```

Comment:
We might want to handle exceptions from the Inference API (e.g. when a model is overloaded)